### ISSUE TITLE
Missing GitHub project

### ISSUE THREAD

**Issue by glm4** [rootstrap/mobx-session#15]
As per Open Source improvements, GitHub project should be used to professionalize maintenance.

**Comment by francocorreasosa** [comment]
I don't think this is something someone without permissions in the repo can solve, is it?
